ansible/configs/open-training-novelloshell/env_vars.yml

```diff
@@ -0,0 +1,130 @@
+---
+###### VARIABLES YOU SHOULD CONFIGURE FOR YOUR DEPLOYEMNT
+###### OR PASS as "-e" args to ansible-playbook command
+
+# This is an account that must exist in OpenStack.
+# It is used to create projects, access, Heat templates
+admin_user: opentlc-mgr
+
+# Authenication credentials for OpenStack in order to create the things.
+# These should be included with your secrets, but are listed here for reference
+# osp_auth_url:
+# osp_auth_username:
+# osp_auth_password:
+# osp_auth_cloud:
+# osp_auth_project_domain: #usually set to "default"
+# osp_auth_user_domain: #usually set to "default"
+
+# Authenication credentials for IBM Cloud Storage in order to download images from.
+# These should be included with your secrets, but are listed here for reference
+#ibm_api_key: ""
+#ibm_bucket_name: ""
+#ibm_endpoint: ""
+#ibm_auth_endpoint: "https://iam.cloud.ibm.com/identity/token"
+#ibm_resource_id: ""
+
+# Host used to convert images from IBM cloud and upload to Glance
+import_host: 169.47.25.34
+
+# The output_dir holds all of the files generated during the deployment
+# This includes generated Heat templates, SSH config, SSH keys
+# This must be an absolute path and no vars (like $HOME or ~)
+output_dir: /tmp/output_dir
+
+# The name of the agnosticd config to deploy
+env_type: open-training-novelloshell
+
+# The {{ guid }} is used everywhere and it is what differentiates otherwise
+# identical environments. Make this unique. Usually they are 4 characters, but
+# it can be any reasonablre length.
+guid: mydefault
```

Comment:
please remove or comment. Those variables must be passed as extra vars

see https://github.com/redhat-cop/agnosticd/blob/development/docs/Variables.adoc#core-required-variables